tests/functional/test_summary.py

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+from datetime import timedelta
+from time import sleep
+
+from core.test_run import TestRun
+from utils.iotrace import IotracePlugin, parse_json
+
+runtime = timedelta(seconds=8)
+label = "super_iotracer_test_instance"
+
+
+def test_compare_summary():
+    """
+        title: Compare trace summaries.
+        description: |
+          Verify the summary returned at the end of tracing
+          (output of 'iotrace –S') is the same as the summary read
+          from trace files (output of 'iotrace -M -G').
+        pass_criteria:
+          - Trace summaries are the same.
+    """
+    with TestRun.step("Run tracing for a while"):
+        disk = TestRun.dut.disks[0]
+        output = parse_json(TestRun.executor.run(
```

Comment:
Don't we want to run start trace from iotrace plugin?

Reply:
No, because start iotrace with method returns nothing and also runs in background, so we won't get output. That's why I used customized command.